experiment_1/scripts/FlexMex1_10/FlexMex1_10_preprocessing.py

```diff
@@ -271,23 +139,34 @@ def create_solar_pv_profiles():
 
 def main():
     # update elements
-    update_shortage_file()
-    update_load_file()
-    update_wind_onshore()
-    update_wind_offshore()
-    update_solar_pv()
-    update_link_file()
+    update_shortage_file(data_preprocessed_path)
+    update_load_file(data_preprocessed_path, scalars)
+    update_wind_onshore(data_preprocessed_path, scalars)
+    update_wind_offshore(data_preprocessed_path, scalars)
+    update_solar_pv(data_preprocessed_path, scalars)
+    update_link_file(data_preprocessed_path, scalars)
 
     # create sequences
-    create_load_profiles()
-    create_wind_onshore_profiles()
-    create_wind_offshore_profiles()
-    create_solar_pv_profiles()
+    create_load_profiles(data_raw_path, data_preprocessed_path)
+    create_wind_onshore_profiles(data_raw_path, data_preprocessed_path)
+    create_wind_offshore_profiles(data_raw_path, data_preprocessed_path)
+    create_solar_pv_profiles(data_raw_path, data_preprocessed_path)
```

Comment:
These 4 can be abstracted and may be moved to the preprocessing module as well, as proposed in #14. I promised that I would do so. For better clarity, I suggest to address it in a separate PR.

Reply:
Or did you intend to move them already, as you changed the signature?

Reply:
At that point I missed to separate create and update functions. Because this was already started, should we just move them in this PR, too?

Reply:
Yes, let's move it.